### DISCUSSION TITLE
GTC-NRTR比SAR难收敛

### DISCUSSION THREAD

**Discussion by tengxniu** [PaddlePaddle/PaddleOCR#12197] [Q&A]
同样的数据集使用GTC-NRTR训练，不收敛，而且训练acc不升反降，非常不稳定，难收敛。SAR没有这样的问题

**Comment by QQQTAO** [comment]
> 

想咨询一下你是用的ocrv4识别的service配置文件训练的吗，我batch 训练数据120万左右，训练200个epoch，准确度0.844，ctcloss下降到1.5左右，NRTRLoss几乎没怎么下降，你的训练数据量有多大，最后损失下降到呀，能不能发一下的训练情况参考一下

**Comment by tengxniu** [comment]
> > 
> 
> 想咨询一下你是用的ocrv4识别的service配置文件训练的吗，我batch 训练数据120万左右，训练200个epoch，准确度0.844，ctcloss下降到1.5左右，NRTRLoss几乎没怎么下降，你的训练数据量有多大，最后损失下降到呀，能不能发一下的训练情况参考一下

轻量版：200w数据，ctcloss:0.8, NRTRLoss:1.3
